lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc

```diff
@@ -0,0 +1,90 @@
+= Booleans and Filters (2)
+
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+/* Cram everything to the top instead of distributing space equally */
+#content { display: block !important; }
+</style>
+++++
+
+== Booleans and Numbers
+
+In the @starter-file{boolean}, open the Transformer called `filter-is-???`, pictured below. For each prompt below, you will select "Edit" in the Transformer, and then enter the specified Boolean expression. (Relevant boxes are highlighted in red in the image on the right.)
+
+[cols="4, 10", frame="none", grid="none", stripes="none"]
+|===
+
+
+|
+@image{../images/filter-is-blank.png, 300}
+
+|
+
+@n Click Edit. Change `Pounds=` so that it says `Pounds=32`. What happened? @fitb{}{}
+
+@fitb{}{}
+
+@vspace{1ex}
+
+@n What would be a good name for a Transformer with the expression `Pounds<32`?
+
+@fitb{}{}
+
+@vspace{1ex}
+
+
+@n What would be a good Purpose Statement for a Transformer with the expression `Pounds<32`? @fitb{}{}
+
+|===
+
+@n With your partner, test out each of the Booleans listed on the table below, using `Pounds` @fitb{1em}{} `32` as the expression. Record the effect of each Boolean in the table below.
```

Comment:
The formatting here feels a little cumbersome to me -- I had to read the instructions and look at the table twice to understand what you meant! I bet you can use the same amount of space by just giving each Boolean expression it's own `@n` number, and one long `@fitb` for kids to write the answer